app/assets/stylesheets/components/_main_layout.sass

```diff
@@ -0,0 +1,32 @@
+$Main-background: $Theme-white
+$Main-color: $Theme-blue
+$Main-height-header: $Theme-height-header
+$Main-height-footer: $Theme-height-footer
+
+.MainLayout
+  display: flex
+  flex-direction: column
+  height: 100%
+  text-align: center
+
+.MainLayout-header
+  flex-shrink: 0
+  height: $Main-height-header
+  background: $Main-background
+  color: $Main-color
```

Comment:
No, i do not. Don't we need to have a body component? I mean, the color for the header and footer may have to be set on header and footer component, respectively.